src/views/Home.vue

```diff
@@ -19,27 +19,31 @@ interface RouteDefinition {
 export default class Home extends Vue {
   private routes: RouteDefinition[] = [
     {
-      title: 'aesthetics',
+      title: 'Indian Music Experiment',
```

Comment:
The title here should be the key in i18n file to make it translation compatible.